<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::FontBrowserWidget</class>
 <widget class="QWidget" name="GammaRay::FontBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="fontSearchLine">
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="fontTree">
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="fontText">
         <property name="text">
          <string>The quick brown fox jumps over the lazy dog</string>
         </property>
         <property name="placeholderText">
          <string>Enter some text here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="fontOptions" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Point size:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pointSize"/>
          </item>
          <item>
           <widget class="QCheckBox" name="boldBox">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Bold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="italicBox">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Italic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="underlineBox">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Underline</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="GammaRay::DeferredTreeView" name="selectedFontsView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
